pkgs/by-name/my/mysql-workbench/package.nix

```diff
@@ -3,49 +3,52 @@
   stdenv,
   fetchurl,
   replaceVars,
+
   cmake,
   ninja,
   pkg-config,
-  glibc,
-  gtk3,
-  gtkmm3,
-  pcre,
+  jre,
   swig,
-  antlr4_12,
+  wrapGAppsHook3,
+
+  bash,
+  coreutils,
+  glibc,
   sudo,
+
+  cairo,
   mysql,
+  libiodbc,
+  proj,
+
+  antlr4_12,
+  gtkmm3,
   libxml2,
   libmysqlconnectorcpp,
   vsqlite,
   gdal,
-  libiodbc,
-  libpthreadstubs,
-  libXdmcp,
-  libuuid,
-  libzip,
-  libsecret,
-  libssh,
-  python3,
-  jre,
   boost,
-  libsigcxx,
-  libX11,
+  libssh,
   openssl,
   rapidjson,
-  proj,
-  cairo,
-  libxkbcommon,
-  libepoxy,
-  wrapGAppsHook3,
-  at-spi2-core,
-  dbus,
-  bash,
-  coreutils,
-  zstd,
+  libuuid,
+  libzip,
+  libsecret,
+  python3Packages,
 }:
 
 let
-  inherit (python3.pkgs) paramiko pycairo pyodbc;
+  # for some reason the package doesn't build with swig 4.3.0
+  swig_4_2 = swig.overrideAttrs (prevAttrs: {
+    version = "4.2.1";
+    src = prevAttrs.src.overrideAttrs {
+      outputHash = "sha256-VlUsiRZLScmbC7hZDzKqUr9481YXVwo0eXT/jy6Fda8=";
+    };
+  });
```

Comment:
eval doesn't like this

Reply:
Ah, I see that `fetchFromGitHub` uses `// { inherit rev; }` to put the `rev` value onto the derivation...
and `overrideAttrs` destroys that.
I guess I'll use the intended method: `.override`